test/module/irohad/consensus/yac/peer_orderer_test.cpp

```diff
@@ -42,17 +43,12 @@ class YacPeerOrdererTest : public ::testing::Test {
     orderer = PeerOrdererImpl(pbfactory);
   }
 
-  std::vector<std::shared_ptr<shared_model::interface::Peer>> peers = [] {
+  std::vector<wPeer> peers = [] {
     std::vector<std::shared_ptr<shared_model::interface::Peer>> result;
     for (size_t i = 1; i <= N_PEERS; ++i) {
-      auto peer = std::make_shared<MockPeer>();
-      EXPECT_CALL(*peer, address())
-          .WillRepeatedly(ReturnRefOfCopy(std::to_string(i)));
-      EXPECT_CALL(*peer, pubkey())
-          .WillRepeatedly(
-              ReturnRefOfCopy(shared_model::interface::types::PubkeyType(
-                  std::string(32, '0'))));
-
+      auto peer = makePeer(
```

Comment:
Since you have modified that, could you please rename N_PEERS variable?

kPeers or kPeersQuantity ?

Reply:
Should not it be at least a constant?